### ISSUE TITLE
Disk Manipulator Empty ItemStack Issue

### ISSUE THREAD

**Issue by Dykam** [raoulvdberge/refinedstorage#1182]
#### Issue description:
Empty Air Itemstacks show up in the system, which is generally harmless but can cause issues like `java.lang.IllegalArgumentException: Stack can not be invalid`.

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.11.2
- Forge: 13.20.0.2282
- Refined Storage:  1.4.3

#### If a (crash)log is relevant for this issue, link it here:

https://gist.github.com/Dykam/cfe468875197b7a5d889c68ba8729782
https://gist.github.com/Dykam/c37e0f780fe9bfe3262959476f5c5b0c

#### More details

https://github.com/raoulvdberge/refinedstorage/issues/1126#issuecomment-292027571

> I've just experienced a similar issue, there is an empty air itemstack stored. Hovering (tooltip) worked fine, however searching causing only the empty stack to be left caused a crash: `OreDictionary.getOreIDs` "Stack can not be invalid!".
> 
> https://github.com/raoulvdberge/refinedstorage/blob/aba639c/src/main/java/com/raoulvdberge/refinedstorage/gui/grid/stack/GridStackItem.java#L72 -> https://github.com/MinecraftForge/MinecraftForge/blob/04885b3  /src/main/java/net/minecraftforge/oredict/OreDictionary.java#L440
> 
> Crashlogs https://gist.github.com/Dykam/cfe468875197b7a5d889c68ba8729782
> 
> Upon more testing, it disappeared after playing with the disks inserted. Found out it's one specific disk. When it's inserted, the air item appears. Reopening the grid multiplies the item, flooding it if you keep going. Removing gets rid of them all, reinserting shows one air item again.
> 
> After swapping the disks around a bit the issue disappeared entirely... I have a slight suspicion that something bugged out after "Level Up!" mined default ore blocks with an NBT tag, which weren't picked up by the processing crafting patterns made using default ores.
> 
> tl;dr couldn't reproduce, hope here is enough information to figure it out.

https://github.com/raoulvdberge/refinedstorage/issues/1126#issuecomment-296442143

> Adding another crashlog https://gist.github.com/Dykam/c37e0f780fe9bfe3262959476f5c5b0c It seems to be reoccuring, but this time it bricked my world. Somehow Itemstack(0, 0) keeps showing up in the system. But I'm not entirely sure how to trace that back to a cause. Though it sounds like something which RS shouldn't even let in even if an external mod tries to insert it.

**Comment by raoulvdberge** [comment]
I removed some irrelevant parts of your issue description, since the crash you were getting for this particular issue had nothing to do with empty itemstacks *in* the system.
